Source/WebCore/platform/network/create-http-header-name-table

```diff
@@ -137,21 +137,20 @@ bool findHTTPHeaderName(StringView stringView, HTTPHeaderName& headerName)
         return false;
 
     if (stringView.is8Bit()) {
-        if (auto nameAndString = HTTPHeaderNamesHash::findHeaderNameImpl(reinterpret_cast<const char*>(stringView.characters8()), length)) {
+        if (auto nameAndString = HTTPHeaderNamesHash::findHeaderNameImpl(reinterpret_cast<const char*>(stringView.span8().data()), length)) {
             headerName = nameAndString->headerName;
             return true;
         }
     } else {
-        LChar characters[maxHTTPHeaderNameLength];
-        for (unsigned i = 0; i < length; ++i) {
-            UChar character = stringView.characters16()[i];
+        char characters[maxHTTPHeaderNameLength];
```

Comment:
Should this use `std::array`? Probably not that beneficial since it won’t know the length at compile time. Would be nice if we had a `std::span` with the correct length instead of passing a pointer/length pair. Almost want to just make a `std::array` and a `std::span` of the correct length up front, but that’s not really all that great.